db/migrate/20200429045956_remove_bookmarks_with_reminder_post_menu_item.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class RemoveBookmarksWithReminderPostMenuItem < ActiveRecord::Migration[6.0]
+  def up
+    post_menu = SiteSetting.post_menu
```

Comment:
We need to use manual SQL in migrations rather than Ruby objects.

Can you adjust this to update the value in the DB instead?

Reply:
Argh sorry @eviltrout this is just an ingrained habit, I keep forgetting. Will change before merging.